src/main/java/core/basesyntax/service/serviceimpl/FruitFileReaderImpl.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax.service.serviceimpl;
+
+import core.basesyntax.exception.NoFileToReadException;
+import core.basesyntax.service.interfaces.FruitFileReader;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+
+public class FruitFileReaderImpl implements FruitFileReader {
+    private CheckFilePathImpl filePathExist = new CheckFilePathImpl();
+
+    @Override
+    public List<String> readFile(String filePath) {
+        if (filePathExist.checkFilePath(filePath)) {
+            throw new NoFileToReadException("File path is empty cannot read data");
+        }
```

Comment:
this is redundant, there is a try-catch for that which will handle this situation